code/game/gamemodes/cult/runes.dm

```diff
@@ -484,15 +484,18 @@ structure_check() searches for nearby cultist structures required for the invoca
 
 	if(locate(/obj/singularity/narsie) in GLOB.poi_list)
 		for(var/M in invokers)
-			to_chat(M, "<span class='warning'>Nar-Sie is already on this plane!</span>")
-		log_game("Summon Nar-Sie rune failed - already summoned")
+			to_chat(M, "<span class='warning'>[src] fizzles uselessly, Nar-Sie is already on this plane!</span>")
+		playsound(src, 'sound/items/welder.ogg', 50, 1)
+		log_game("Summon Nar-Sie rune at [COORD(src)] failed - already summoned")
 		return
 	//BEGIN THE SUMMONING
 	used = TRUE
 	..()
 	sound_to_playing_players('sound/effects/dimensional_rend.ogg')
 	var/turf/T = get_turf(src)
 	sleep(40)
+	if(locate(/obj/singularity/narsie) in GLOB.poi_list)
```

Comment:
Just to avoid people skimming the code from going "REEEE YOU ALREADY CHECK THIS", what if another nar-sie is created in that 4 seconds?